src/mscorlib/src/System/Decimal.cs

```diff
@@ -346,9 +346,6 @@ public static Decimal Add(Decimal d1, Decimal d2)
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         private static extern void FCallAddSub(ref Decimal d1, ref Decimal d2, byte bSign);
         
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        private static extern void FCallAddSubOverflowed(ref Decimal d1, ref Decimal d2, byte bSign, ref bool overflowed);
```

Comment:
For these fcalls, is there corresponding code in the runtime that can/should be deleted as well?